types/frisby/index.d.ts

```diff
@@ -1,75 +1,234 @@
-// Type definitions for Frisby v0.8.5
+// Type definitions for Frisby 2.0
 // Project: https://github.com/vlucas/frisby
-// Definitions by: Johnny Li <https://github.com/johnny4753>
+// Definitions by: Christopher E. Woodland <https://github.com/cwoodland>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.6
 
-///<reference types="jasmine"/>
+/// <reference types='jasmine'/>
```

Comment:
Does this declaration file itself actually depend on `jasmine`? Do you want consumers of this library to also get all the members of `jasmine` in their global namespace? Or do you just want to be able to use it in the tests?

Reply:
It is my understanding that frisby.js is dependent on jasmine.
I believe I want consumers of this library to also get members of jasmine.
Also, yes my test file relies on jasmine.

Example code from frisby.js: https://github.com/vlucas/frisby/blob/master/src/frisby/spec.js#L330